<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileChooserDialog</class>
 <widget class="QDialog" name="FileChooserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File Chooser</string>
  </property>
  <widget class="QTabWidget" name="fileChooser">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>864</width>
     <height>417</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="recentTab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>Recent</string>
    </attribute>
    <widget class="QListView" name="recentView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>863</width>
       <height>413</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="documentTab">
    <attribute name="icon">
     <iconset resource="CalligraMobile.qrc">
      <normaloff>:/images/48x48/Document.png</normaloff>:/images/48x48/Document.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Document</string>
    </attribute>
    <widget class="QListView" name="documentView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>863</width>
       <height>339</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="presentationTab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="icon">
     <iconset resource="CalligraMobile.qrc">
      <normaloff>:/images/48x48/Presenter.png</normaloff>:/images/48x48/Presenter.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Presentation</string>
    </attribute>
    <widget class="QListView" name="presentationView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>863</width>
       <height>339</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="spreadsheetTab">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="icon">
     <iconset resource="CalligraMobile.qrc">
      <normaloff>:/images/48x48/SpreadSheet.png</normaloff>:/images/48x48/SpreadSheet.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Spreadsheet</string>
    </attribute>
    <widget class="QListView" name="spreadsheetView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>862</width>
       <height>339</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>851</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item row="0" column="1">
     <widget class="QPushButton" name="levelUp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="CalligraMobile.qrc">
        <normaloff>:/images/64x64/VirtualKeyBoard/change.png</normaloff>:/images/64x64/VirtualKeyBoard/change.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="filePath">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>740</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>740</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="CalligraMobile.qrc"/>
 </resources>
 <connections/>
</ui>
